irods/test/login_auth_test.py

```diff
@@ -326,6 +348,41 @@ def test_8(self):
         self.tst0 ( ssl_opt = False, auth_opt = 'pam'    , env_opt = True  )
 
 
+class TestWithSSL(unittest.TestCase):
+    '''
+    The tests within this class should be run by an account other than the
+    service account.  Otherwise there is risk of corrupting the server setup.
+    '''
+
+    def setUp(self):
+        if os.path.expanduser('~') == '/var/lib/irods':
```

Comment:
same question.

Reply:
Acknowledged, difficult to say.  the `login_auth_test.py` script needs to be run in two phases, once by the service account and once by a normal user.  Currently it only makes sense only  on a single node iRODS with a package install  and a service account.

Reply:
okay, that's fine for now then - let's just leave it as is - this is just testing code.